wordpress-nginx/roles/wordpress/tasks/main.yml

```
@@ -13,10 +16,12 @@
   user: name=wordpress group=wordpress home=/srv/wordpress/
 
 - name: Fetch random salts for WordPress config
-  local_action: command curl https://api.wordpress.org/secret-key/1.1/salt/
+  get_url:
+    url: https://api.wordpress.org/secret-key/1.1/salt/
   register: "wp_salt"
   become: no
   become_method: sudo
+  changed_when: false
```

Why?

---

`ansible-lint` error here was
```
[301] Commands should not change things if nothing needs doing
wordpress-nginx/roles/wordpress/tasks/main.yml:15
Task/Handler: Fetch random salts for WordPress config
```

---

I'd argue that it should be
```suggestion
  changed_when: true
```